### ISSUE TITLE
Isofox -excluded_gene_id_file: Expected behavior or bug?

### ISSUE THREAD

**Issue by toddajohnson** [hartwigmedical/hmftools#273]
As an attempt to remove rRNA transcripts from my RNA-seq analysis, I passed a file of gene symbols to Isofox using -excluded_gene_id_file.

The rRNA genes are very small and sometimes embedded in introns of coding genes, and I just noticed that some protein coding genes were missing in the Isofox output, but  when I reran it without passing the file to exclude those genes, they appeared in the output..  Is that expected behavior?

**Comment by charlesshale** [comment]
In the current implementation, the genes listed in that file are excluded but so are any genes which overlap them. Does that explain the result you're getting?

Any alternative would be to add to the list of existing 7 enriched gene IDs (see read-me for which they are), and these will then be handled more efficiently and their impact on TPM diminished (again as described in the read-me).

**Comment by toddajohnson** [comment]
Hi Charles,
Thank you for the information.  That confirms what I thought was occurring behind the scenes.
Since I first tried -excluded_gene_id_file, I have added a step before running STAR to bin read pairs in the fastq  based on whether or not they match fastas of known "abundant" sequences; basically mt genome and rRNA genes.  That reduced my problems considerably, as some of our samples have lots of mitochondria.
